docs/integrations/app-development/opentelemetry/jmx-opentelemetry.md

```diff
@@ -0,0 +1,214 @@
+
+  Make sure the see the opentelemetry-java-contrib-jmx-metrics version is supported in the opentelemetry version you are using by going through the [releases](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases) section  and searching for jmxreceiver in that page or in [supported jars code](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/jmxreceiver/supported_jars.go#L33).
+
+  JMX Metric Gatherer metric extension supports Java 8+, though SASL is only supported where com.sun.security.sasl.Provider is available.
+
+
+2. Configure your java application to enable jmx
+
+  In most cases you need to set below system variables in your applications's startup script
+
+  ```-Dcom.sun.management.jmxremote.port=11099
+  -Dcom.sun.management.jmxremote.authenticate=false
+  ```
+
+  If you want to set up authentication then set below system variables
+
+  ```
+  -Dcom.sun.management.jmxremote.authenticate=true
+  -Dcom.sun.management.jmxremote.password.file=<application conf directory>/jmx.password
+  -Dcom.sun.management.jmxremote.access.file=<application conf directory>/jmx.access
+  ```
+
+  Below are docs for popular java applications:
+
+    * [Active MQ](https://activemq.apache.org/jmx)
+    * [Kafka](https://kafka.apache.org/documentation/#remote_jmx)
+    * [Apache Tomcat](https://tomcat.apache.org/tomcat-9.0-doc/monitoring.html#Enabling_JMX_Remote)
+    * [Cassandra](https://cassandra.apache.org/doc/4.1/cassandra/operating/security.html#jmx-access)
+    * [HBase](https://hbase.apache.org/metrics.html)
+    * [Solr](https://solr.apache.org/guide/8_3/using-jmx-with-solr.html#configuring-jmx)
+    * [Zookeeper](https://zookeeper.apache.org/doc/r3.4.2/zookeeperJMX.html#ch_starting)
+
+
+
+## Collecting Metrics and Installing App for JMX
+
+The process to set up collection for JMX Metrics is done through the following steps.
+
+### Step 1: Set up OpenTelemetry Collector
+
+{@import ../../../reuse/apps/opentelemetry/set-up-collector.md}
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Kafka-OpenTelemetry/Kafka-Collector.png' alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for JMX Collection.
+
+Below is the input required:
+
+- **JMX Endpoint**: The URL of the broker endpoint (default: `localhost:9092`).
+- **Jar File Path**: Enter the path to the OpenTelemetry JMX Metric Gatherer file configured in the prerequisites section.
+- **UserName**: Username for JMX authentication, if applicable.
+- **Password**: Password for JMX authentication, if applicable.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<!-- <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Kafka-OpenTelemetry/Kafka-YAML.png' alt="YAML" /> -->
```

Comment:
do you have the JMX version of this image?

Reply:
I will update the image for this